gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/FileExporter.java

```diff
@@ -110,7 +109,7 @@ private String zipTargetNameToJsonName(String targetName) {
     }
 
     public String makeDestinationMinioPath(OffsetDateTime offsetDateTime, FileKind filekind) {
-        ZonedDateTime targetDateTime = offsetDateTime.atZoneSameInstant(ZoneId.of(ZONE_ID));
+        ZonedDateTime targetDateTime = offsetDateTime.atZoneSameInstant(ZoneId.systemDefault());
```

Comment:
I think it's better to add ZoneId to swe runner configuration

Reply:
Sure will do !